File: test_data/code/python/1360-number-of-days-between-two-dates.py
Python
from datetime import datetime

class Solution:
    def daysBetweenDates(self, date1: str, date2: str) -> int:
        dt1 = datetime.strptime(date1, '%Y-%m-%d')
        dt2 = datetime.strptime(date2, '%Y-%m-%d')
        
        return abs((dt2 - dt1).days)

File: test_data/code/java/1360-number-of-days-between-two-dates.java
import java.time.LocalDate;
import java.time.temporal.ChronoUnit;

class Solution {
    public int daysBetweenDates(String date1, String date2) {
        LocalDate localDate1 = LocalDate.parse(date1);
        LocalDate localDate2 = LocalDate.parse(date2);
        long days = ChronoUnit.DAYS.between(localDate1, localDate2);
        return (int) days;
    }
}